fabrictestbed/cli/cli.py

```diff
@@ -315,12 +330,16 @@ def query(ctx, cmhost: str, ochost: str, tokenlocation: str, projectid: str, sco
         slice_manager = __get_slice_manager(cm_host=cmhost, oc_host=ochost, project_id=projectid, scope=scope,
                                             token_location=tokenlocation)
 
-        slice_object = Slice()
-        slice_object.slice_id = sliceid
+        status, response = slice_manager.slices(slice_id=sliceid)
+        if status != Status.OK:
+            click.echo(f'Query Sliver(s) failed: {status.interpret(exception=response)}')
+            return
+
+        slice_object = response[0]
         status, response = slice_manager.slivers(slice_object=slice_object)
 
         if status == Status.OK:
-            click.echo(response)
+            click.echo(json.dumps(list(map(lambda i: __unpack(i.to_dict()), response)),indent=2))
```

Comment:
Same observation as above, regarding line 172: MyPy complained about this `map`. Probably doesn't matter.